### DISCUSSION TITLE
hey,everyone there is some proble with python3  it is installed still throe ana error !#user/bin/python3

### DISCUSSION THREAD

**Discussion by anshugarg401** [Cyfrin/foundry-full-course-cu#1647] [General]
.

**Comment by EngrPips** [comment]
Hello @anshugarg401, Did you resolve this issue already?

**Reply by anshugarg401** [reply]
No

**Reply by anshugarg401** [reply]
![1000026378.jpg](https://github.com/Cyfrin/foundry-full-course-f23/assets/92512766/067f2ade-85bd-4cc8-ad70-72560ada5784)

**Reply by EngrPips** [reply]
@anshugarg401, Please share the code you are trying to deploy. Seem like you missed pragma declaration.

**Reply by anshugarg401** [reply]
![IMG20240426205010.jpg](https://github.com/Cyfrin/foundry-full-course-f23/assets/92512766/3fe0a240-3471-40d2-a9ea-385e937aab2e)

This is the code

**Reply by EngrPips** [reply]
Use `import {Script} from "forge-std/Script.sol"` instead of `import {Scripts} from "../lib/forge-std/Scripts/vm.py"` and also change the name of the Script from `Script`  to  `DeployOurToken.` so your Script should like the below.

```javascript
//SPDX-License-Identifier:MIT
pragma solidity ^0.8.18

import {Script} from "forge-std/Script.sol";
import {MyToken} from "../src/MyToken.sol";

contract DeployOurToken is Script {
// all your logic here !!!!!!
}
```

**Reply by anshugarg401** [reply]
Thanks !!! 😊 It's resolved now

**Reply by EngrPips** [reply]
Anytime friend, Keep crushing it.
